src/main/java/com/kainos/ea/model/CompetencyRequest.java

```diff
@@ -0,0 +1,46 @@
+package com.kainos.ea.model;
+
+public class CompetencyRequest {
+
+
+    private String competencyName;
+    private String description;
+    private String bandName;
+
+
+    public CompetencyRequest(String competencyName, String description, String bandName) {
```

Comment:
Should this have the @JSONProperty annotation?

```suggestion
    public CompetencyRequest(String competencyName, String description, String bandName) {
```